src/idsk/components/header-web/header-web.js

```diff
@@ -119,43 +120,49 @@ HeaderWeb.prototype.handleSubmenuClick = function (e) {
     var $srcEl = e.target || e.srcElement;
     var $toggleButton = $srcEl.closest('.idsk-header-web__nav-list-item');
     var $currActiveItem = this.$module.querySelector('.idsk-header-web__nav-list-item--active');
-
-    if ($currActiveItem) {
+    
+    if($currActiveItem && $currActiveItem.isEqualNode($toggleButton)){
         $currActiveItem.classList.remove('idsk-header-web__nav-list-item--active');
-    }
+    }else{
+        if($currActiveItem){
```

Comment:
taktiez si treba dat pozor na coding standards ako:
`if(xyz){` => `if (xyz) {`, `}else{` => `} else {`
ale to su len detaily